tests/api/EventFunnelCest.php

```diff
@@ -45,17 +44,28 @@ public function showStatsByUserId(ApiTester $I)
                 'skip_call_booking' => 20,
                 'thankyou_page' => 20,
                 'leave' => 20,
-                'sign up' => 20
+                'sign up' => 20,
+                'summarization' => [
+                    'visit_mainpage' => $response['counters']['summarization']['visit_mainpage'],
```

Comment:
There should be numbers, not $response['counters']['summarization']['visit_mainpage']

You should compare the expected behavior of statistics with the real behavior, real response to detect issues in your code, and to test yourself.